Services/AccessControl/classes/class.ilRbacAdmin.php

```diff
@@ -347,14 +344,15 @@ public function deassignUser($a_rol_id,$a_usr_id)
 
 		include_once('Services/LDAP/classes/class.ilLDAPRoleGroupMapping.php');
 		$mapping = ilLDAPRoleGroupMapping::_getInstance();
-		$mapping->deassign($a_rol_id,$a_usr_id); 
-		
-		$ref_id = $GLOBALS['rbacreview']->getObjectReferenceOfRole($a_rol_id);
-		$obj_id = ilObject::_lookupObjId($ref_id);
-		$type = ilObject::_lookupType($obj_id);
-		
-		ilLoggerFactory::getInstance()->getLogger('ac')->debug('Raise event deassign user');
-		$GLOBALS['ilAppEventHandler']->raise(
+		$mapping->deassign($a_rol_id,$a_usr_id);
+
+		if ($res) {
```

Comment:
This is only to check if the user had to be unassigned. Only in this case an event makes sense